src/integration/pages/homePage/mostReadTests.js

```diff
@@ -0,0 +1,14 @@
+export default () => {
+  describe('mostRead', () => {
+    const mostRead = document.querySelector(`[data-testid^="most-read"]`);
+
+    it('Is displayed', () => {
+      expect(mostRead).toBeInTheDocument();
+      expect(mostRead).toBeTruthy();
+    });
+
+    it('has a link', () => {
+      expect(mostRead.querySelector('a')).toMatchSnapshot();
+    });
```

Comment:
I wonder if we could get this to check all the links?

I would like to do it in a similar way to https://github.com/bbc/simorgh/blob/28185d1173eeeff477dc363b6f0b7d95f056c17d/src/integration/common/appleItunesAppBanner.js

```suggestion
    it('has multiple items', () => {
      const mostReadLinks = mostRead.querySelectorAll('a');
      
      mostReadLinks.forEach(link, () => {
        const url = link.getAttribute('href');
        const text = link.textContent;
        
        expect({ url, text }).toMatchSnapshot();
      };
    });
```